test/dump/local-indices.txt

```diff
@@ -0,0 +1,60 @@
+;;; TOOL: run-objdump
+;;; ARGS0: -v
```

Comment:
Do we need `-v` here?  Maybe keep the test smaller by removing it?

Reply:
Smaller sounds good to me, it's gone!